modules/local/rename_align_files.nf

```diff
@@ -0,0 +1,30 @@
+process RENAME_ALIGN_FILES {
```

Comment:
I'm a bit surprised that it is necessary to use this module. It seems to be preparing input for `SMNCOPYNUMBERCALLER`. Does `SMNCOPYNUMBERCALLER` require the bam+bai to be named in a certain way?  Perhaps this can be avoided in a future release? (Not 2.0)

Reply:
Yeah, I am not happy with this solution either, and it is something I hope to fix in a future release 😅 
Issue is smncopynumbercaller uses the bam file prefix as the samplename, so currently in the outputfile they are of the form samplename_sorted_md instead of samplename.